src/python/zquantum/core/measurement_test.py

```diff
@@ -586,12 +586,17 @@ def test_get_expectation_values_from_measurements(self):
         )
         ising_operator = IsingOperator("10[] + [Z0 Z1] - 10[Z1 Z2]")
```

Comment:
Maybe I am a bit paranoid here but would you mind making this test a bit less symmetric? I would add smth like 110 and 011 to the measurements and pick an Isining operator with other coefficients, e.g. 10, 2, -3. I remember a few cases when my buggy code passed symmetric tests and I spent countless hours hunting down those bugs later...

Reply:
Updated these tests in c1eabba0f84208cfdd63223a61b481a807c6ec2c